src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -5850,9 +5871,9 @@ public override BoundNode VisitAttribute(BoundAttribute node)
             return null;
         }
 
-        public override BoundNode VisitDeconstructValuePlaceholder(BoundDeconstructValuePlaceholder node)
+        public override BoundNode VisitExpressionWithNullability(BoundExpressionWithNullability node)
         {
-            SetNotNullResult(node);
+            ResultType = TypeWithState.Create(node.Type, node.NullableAnnotation == NullableAnnotation.Annotated ? NullableFlowState.MaybeNull : NullableFlowState.NotNull);
```

Comment:
Note: this will need to be adjusted when features/nullable-api is merged (hopefully later today).